src/player.cpp

```diff
@@ -3073,15 +3058,32 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
             fold_and_print( w_info, 0, 1, FULL_SCREEN_WIDTH - 2, c_magenta, s );
             wrefresh(w_info);
 
+            // which of the four paired body parts should get combined?
+            // TODO deduplicate this with print_encumbrance()
+            bool combined_limbs[4] = {}; // all false
+            for (size_t i = 0+4; i<12-1; i+=2) {
```

Comment:
They are magic values. There actually is an enum that turns body part names into enumvals though. I could use that. But that would still leave a lot of code in various places that does arithmetic with the values, hard coded for the current exactly 12 body parts in the current order.